src/llmcompressor/modifiers/quantization/calibration.py

```diff
@@ -147,7 +147,6 @@ def update_weight_global_scale(module: Module):
         should_calculate_gparam=True,
         should_calculate_qparams=False,
     )
-    module.weight_observer.reset()
```

Comment:
Because we only attach one observer, I’m fairly sure we’re resetting to prevent global scale  metrics from impacting quant scale metrics

Reply:
> Update calculate_gparam to restore original running values, rather than relying on resetting after calculation

Reset is replaced by patching and restoring the metrics